tests/pytest/common.py

```diff
@@ -14,3 +17,23 @@ def handler(signum, frame):
     finally:
         signal.setitimer(signal.ITIMER_REAL, 0)
         signal.signal(signal.SIGALRM, signal.SIG_DFL)
+
+def envMem(env):
+    pid = env.envRunner.masterProcess.pid
+    meminfo = psutil.Process(pid).memory_info()
+    vms = meminfo.vms / 1024
+    rss = meminfo.rss / 1024
+    return {'vsz': vms, 'rss': rss }
+
+def checkEnvMem(env, expected_vsz=None, vsz0=0, threshold=0.1, title=None):
+    if MEMINFO:
+        if title is not None:
+            print(f"--- {title}")
+        pid = env.envRunner.masterProcess.pid
+        print(paella.sh(f'cat /proc/{pid}/status | grep ^Vm', join=False))
+    mem = envMem(env)
```

Comment:
Why not use `info memory`?

Reply:
Do you prefer redis-server internal indications over system ones when investigating memory issues?
In most cases you'll be measuring `vsz` anyway so why not use it in the first place?

Reply:
`info memory` gives you both RSS and the memory Redis think it consumes so I do not see why not use it instead of using `env.envRunner.masterProcess.pid` which is internal to RLTest and might change someday and break this test.

Reply:
Ok.